README.md

```diff
@@ -71,7 +71,7 @@ $ cd ~/gnu/github/exiv2  # location of the project code
 $ mkdir build && cd build
 $ cmake .. -DCMAKE_BUILD_TYPE=Release
 $ cmake --build .
-$ make tests
+$ make test
```

Comment:
I suggest we use the same `ctest` command used in CI - there should be no explicit dependence on autotools.

Reply:
I agree.  Let's avoid talking about make and autotools magic.